### PR TITLE
Docstring rampage

### PR DESCRIPTION
**Pull request by DanHickstein**
I made a whole bunch of changes for the sake of improving the docs. I think that they are in pretty good shape now. 

Another big change was to comment out 
`abel.tools.symmetry.center_image` and `abel.tools.symmetry.center_image_asym`. 

I am now fairly comfortable with the interpolation (`shift`) based method, and so I don't think that these are needed anymore.

After this gets merged, it would be great if another set of eyes could proofread the docs (@stggh and @DhrubajyotiDas). After that, I think Version 0.7 is nearly out the door!

### CODE REVIEW

**Comment by DanHickstein** [comment]
By the way, if we want to check if `s` is a string and have it be compatible with both Python 2 and Python 3, we need to use something like this:
```
from six import string_types
if isinstance(s, string_types):
    #do stuff
```

http://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility

File: abel/basex.py [hunk 1]
```diff
@@ -67,8 +69,8 @@ def basex_transform(data, nbf='auto', basis_dir='./', dr=1.0, verbose=True,
         should correspond to the central column of the image. 
     nbf : str or list
         number of basis functions. If ``nbf='auto'``, it is set to ``(n//2 + 1)``.
-        This is what you should use,
-        since basex does not work reliably in other situations
+        *This is what you should alwasy use*,
```
Comment (DhrubajyotiDas) [inline abel/basex.py]: Typo here - `always`

File: abel/tools/symmetry.py [hunk 2]
```diff
@@ -26,82 +26,70 @@ def get_image_quadrants(IM, reorient=True, symmetry_axis=None,
...
+    
+        (0) symmetry_axis = None
+        
+            returned image   Q1 | Q0
+                            ----o----
+                             Q2 | Q3
+                        
+        (1) symmetry_axis = 0
+
+            Combine:  Q01 = Q1 + Q2, Q23 = Q2 + Q3
```
Comment (DhrubajyotiDas) [inline abel/tools/symmetry.py]: This should be

`Combine: Q01 = Q0 + Q1, Q23 = Q2 + Q3`

File: abel/transform.py [hunk 3]
```diff
@@ -86,8 +89,8 @@ def transform(
         Default is ``(True, True, True, True)``, which uses all quadrants.
         
     recast_as_float64 : boolean
-        True/False that determines if the input image should be recast to float64. 
-        Many images are imported in other formats (such as ``uint8`` or ``unit15``)
+        True/False that determines if the input image should be recast to ``float64``. 
+        Many images are imported in other formats (such as ``uint8`` or ``unit16``)
```
Comment (DhrubajyotiDas) [inline abel/transform.py]: Should this be ``uint16`` ?

**Comment by DanHickstein** [comment]
Thanks for catching these @DhrubajyotiDas!

Looking at the quadrants code, I think that we also need to change a lot of these docstrings to make it clear that we are averaging, not adding. Maybe @stggh can do this in his next PR.

**Comment by DanHickstein** [comment]
I'll go ahead and merge this.

**Comment by DhrubajyotiDas** [comment]
Great work @DanHickstein!

**Comment by DanHickstein** [comment]
I made some tweaks to the readthedocs.org page, and now the docs are building more completely. Check it out: 
http://pyabel.readthedocs.org/

Looks pretty cool!!

**Comment by DhrubajyotiDas** [comment]
@DanHickstein When I click on the links to the individual transforms on this page
>http://pyabel.readthedocs.org/en/latest/readme_link.html#introduction

they go to the github wiki, instead of 
>http://pyabel.readthedocs.org/en/latest/transform_methods/basex.html

**Comment by DanHickstein** [comment]
Oh, good point, I need to change those!

On Thu, Mar 3, 2016 at 9:22 PM, Dhrubajyoti Das <notifications@github.com>
wrote:

> @DanHickstein <https://github.com/DanHickstein> When I click on the links
> to the individual transforms on this page
>
> http://pyabel.readthedocs.org/en/latest/readme_link.html#introduction
>
> they go to the github wiki, instead of
>
> http://pyabel.readthedocs.org/en/latest/transform_methods/basex.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PyAbel/PyAbel/pull/132#issuecomment-192094251>.
>



-- 
Daniel D. Hickstein
DanHickstein@gmail.com
Phone: 301 787 1853

**Pull request merged**